bridge/client/app/_components/ktb-sli-breakdown/ktb-sli-breakdown.component.ts

```diff
@@ -95,22 +95,13 @@ export class KtbSliBreakdownComponent implements OnInit {
       this.columnNames = ['details', 'name', 'value', 'weight', 'passTargets', 'warningTargets', 'result', 'score'];
     }
     return indicatorResults.map((indicatorResult) => {
-      let accSum = 0;
-      let accCount = 0;
-      for (const comparedIndicatorResult of this.comparedIndicatorResults) {
-        const result = comparedIndicatorResult.find((res) => res.value.metric === indicatorResult.value.metric);
-        if (result) {
-          accSum += result.value.value;
-          accCount++;
-        }
-      }
-      const comparedValue = accSum / accCount;
+      const comparedValue = this.calculateComparedValue(indicatorResult);
       const compared: Partial<SliResult> = {};
-      if (comparedValue) {
+      if (!!comparedValue) {
```

Comment:
This will not work if `comparedValue` is `0`

Reply:
Actually I thought we should not calculate the change if the `comparedValue` is `0`, but of course `0` could also be a regular value.